src/pages/Home/Home.style.ts

```diff
@@ -5,6 +5,9 @@ type Props = {
 };
 
 export const HomeStyle = styled.div<Props>`
-  overflow-y: ${(props) => (props.menuOpen ? "hidden" : "auto")};
+  overflow-y: ${(props) => (props.menuOpen ? "hidden" : "visible")};
   height: ${(props) => (props.menuOpen ? "100vh" : "auto")};
+
+  //css here to prevent scrolling when menu modal is open.
```

Comment:
Issue with scroll.
With the sticky nav bar, I can open the menu while scrolled down. Opening the menu while scrolled down will cause the window to scroll back to the top. Scroll position isn't maintained. 

This css helps to prevent scrolling while modal is open, but also introduces this bug.